config/clients/java/template/Java/git_push.sh.mustache

```diff
@@ -0,0 +1,57 @@
+#!/bin/sh
```

Comment:
delete

Reply:
Removed in [6f9804d](https://github.com/openfga/sdk-generator/pull/158/commits/6f9804d21724edeff04096b82f2a999ab7a75144)